docs/blog/newsletter-april-2018.rst

```diff
@@ -0,0 +1,135 @@
+* Options for auto-generating reference documentation for the APIs, if possible
+* And if auto-generation isn't an option, how will you communicate changes in the APIs to me so I can document them?
+
+Turns out this is a situation that lots of others had dealt with, and the consensus was strong: testing is key. 
+If you want to maintain up-to-date samples, it's well worth putting in the time to build some kind of testing that runs the latest version of the product and the relevant areas of code to the samples in your documentation. 
+Granted, these test can difficult to implement, especially if you're working with pre-existing, manually generated content, but even small steps are steps in the right direction.
+
+*Want to dig deeper into docs testing? We've got just the channel for you:* `#doc-testing <https://writethedocs.slack.com/messages/doc-testing>`_
+
+--------------------------------
+Number formatting in data tables
+--------------------------------
+
+This month in #general, there was a quick but engaging discussion on conventions for formatting numbers in tables and specifically, how best to handle blank cells. 
+The community's answers yielded several helpful guidelines for number and table formatting:
+
+- Avoid leaving cells blank unless the data for the cell is actually missing. 
+  Blank cells often indicate missing data.
+- When you use a placeholder or no-value indicator, use a different alignment than you used for cells with values.
+- If, however, you want to specifically draw attention cells that have no value, leaving them blank can be a good way to get them to stand out in larger tables.
+
+The group also had some good examples of formatting options for no-value indicators:
+
+- ``N/A``
+- ``$0.00``
+- Centered em dash
+- Or a dash length of your choice! ;)
+
+As always, context is key. 
+Consider your audience and publishing format when determining the formatting fate of your empty cells.
+
+*Got a finicky formatting or mechanics question? The friendly folks in* `#general <https://writethedocs.slack.com/messages/general>`_ *can help you answer it.*
+
+------------------------------------------
+New tool to try out: automated screenshots
+------------------------------------------
+
+Although we don't often feature conversations about specific tools, there was one that popped up this month that we thought was worth a quick mention. 
+On what feels like a weekly basis, someone brings up the difficulty of keeping screenshots up to date. 
+So when someone pointed to `CronShot <https://www.npmjs.com/package/cronshot>`_, a Node module for programmatically scheduling and saving screenshots of web pages, we thought we'd share it around. 
+In addition, if you're so inclined (and comfortable with JavaScript), you can add the `Nick.js <https://www.npmjs.com/package/nickjs>`_ scraping library (oh my!) in order to automate browser navigation altogether.
```

Comment:
If a short explanation is even possible, it might be nice to add one for what scraping is in this context.

Reply:
a good call – I went the other direction, though, and eliminated the ref to scraping altogether